test/unit/modules/wallet.test.ts

```diff
@@ -212,13 +222,19 @@ describe('Wallet class', () => {
       };
       const hashInfo = { amount: '2000000000000000000000000000000' };
       wallet.info = jest.fn().mockResolvedValue(info);
+      workMock.generate
+        .mockResolvedValueOnce({ work: 'work' } as any)
+        .mockResolvedValueOnce({ work: 'work2' } as any);
       blocksMock.info.mockResolvedValue(hashInfo as any);
```

Comment:
I would consider `any` bad practice, because the type checking TypeScript provides gets disabled. Maybe you could remove it?